From: Remy Bohmer <linux@bohmer.net>
Subject: utelnetd: add EXTRA_CFLAGS

This patch makes utelnetd use the EXTRA_CFLAGS during compilation
of the package. This is required for some architectures to pass flags like
CPU-type to the compilation.

Signed-off-by: Remy Bohmer <linux@bohmer.net>

---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: utelnetd-0.1.9/Makefile
===================================================================
--- utelnetd-0.1.9.orig/Makefile	2003-08-08 09:25:18.000000000 +0200
+++ utelnetd-0.1.9/Makefile	2007-08-29 12:54:35.000000000 +0200
@@ -35,7 +35,7 @@ OBJS      = utelnetd.o
 CC        = $(CROSS)gcc
 INSTALL   = install

-CFLAGS	 += -I. -pipe -DSHELLPATH=\"$(LOGIN)\" -Wall
+CFLAGS	 += $(EXTRA_CFLAGS) -I. -pipe -DSHELLPATH=\"$(LOGIN)\" -Wall

 ifneq ("","$(DEBUG)")
 CFLAGS   += -DDEBUG -g -Os
@@ -53,7 +53,7 @@ endif
 all: $(PROGS)

 $(PROGS): $(OBJS)
-	$(CC) $(LDFLAGS) $^ $(LDLIBS) -o $@
+	$(CC) $(LDFLAGS) $(EXTRA_CFLAGS) $^ $(LDLIBS) -o $@
 	$(STRIP) --remove-section=.comment --remove-section=.note $@

 .PHONY: install
